server/views/actions-rule-set.njk

```diff
@@ -0,0 +1,56 @@
+<dl class="govuk-summary-list--no-border">
+  {% for rule in rules %}
+    {% set ruleID = item.actionID + "_" + rule.id %}
+    {% set backgroundColour = 'background-color: white' %}
+
+    {% if loop.index % 2 %}
+      {% set backgroundColour = 'background-color: #f3f2f1' %}
+    {% endif %}
+
+    <div id={{ ruleID }} class="govuk-summary-list__row" style="{{ backgroundColour }}">
+      <dt class="govuk-summary-list__key" style="padding-left: 10px">
```

Comment:
That's all just moved from `actions.njk` into a separate file so it could be reused. I didn't even look at the content of the moved code!

Seems reasonable to create a class, although probably not worth fussing over at this stage?

Reply:
No, I felt it was a bit of a pettifogging point when I made it, so leave it for now :)